src/main/resources/com/nirima/jenkins/plugins/docker/DockerTemplate/help-instanceCapStr.html

```diff
@@ -0,0 +1,3 @@
+<div>
+    Max number of instances of this image to run, or 0 for unlimited
```

Comment:
I found this:

```java
        if (instanceCapStr.equals("")) {
            this.instanceCap = Integer.MAX_VALUE;
        } else {
            this.instanceCap = Integer.parseInt(instanceCapStr);
        }
```

The description above is wrong. In my opionion if it's 0 it should not start Instances on this machine. If it's blank, then it's Integer.MAX_VALUE.